org.gnome.Cheese.yml

```diff
@@ -0,0 +1,75 @@
+id: org.gnome.Cheese
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: 3.34
+command: cheese
+tags:
+  - 3.34.0
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --device=all
+  - --metadata=X-DConf=migrate-path=/org/gnome/Cheese/
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+cleanup:
+  - /include
+  - /lib/girepository-1.0
+  - /lib/pkgconfig
+  - /man
+  - /share/aclocal
+  - /share/gir-1.0
+  - /share/pkgconfig
+  - /share/man
+  - /share/gtk-doc
+  - '*.la'
+  - '*.a'
+modules:
+  - name: gnome-desktop
+    buildsystem: meson
+    config-opts:
+      - -Ddebug_tools=false
+      - -Dudev=disabled
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/gnome-desktop.git
+  - shared-modules/lua5.3/lua-5.3.5.json
+  - name: libquvi-scripts
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/quvi/libquvi-scripts-0.9.20131130.tar.xz
+        sha512: 97de4cead30dc181da700cbf3a0ad66ee709fef17f449f75cd43e0376a4aa53c7b2174188130aaa34dcc7ddc578581111f3655cb10f4b0c92df2f75d32c72a02
+  - name: libquvi
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/quvi/libquvi-0.9.4.tar.xz
+        sha512: 287015e370e7b7f98f0f4c3717f5b044c8fcfb8b823c7988cd248e56b5fbbfa1a1aa421fbdafb38d727b79495267ccd4dead71587052117db53f29236268b651
+  - name: totem-pl-parser
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/totem-pl-parser.git
+  - name: totem-video-thumbnailer
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/totem-video-thumbnailer.git
+  - name: gnome-video-effects
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/gnome-video-effects.git
+  - name: cheese
+    buildsystem: meson
+    config-opts:
+      - -Dintrospection=false
+    sources:
+      - type: git
+        tag: 3.34.0
+        url: https://gitlab.gnome.org/GNOME/cheese.git
```

Comment:
Use the available sources at https://download.gnome.org/sources/. The same for all GNOME modules.

Reply:
Should be fixed in https://github.com/flathub/flathub/pull/1320/commits/b185df4fa279eb0bfef9ad620b8e76bf6d70d42a Thanks.

Reply:
*all* modules.

Reply:
I was able to use an archive from https://download.gnome.org/sources/ for all but 1 of the items here. The **totem-video-thumbnailer** repo seems to have been created for Flatpak and doesn't have a matching release over at download.gnome.org. If anyone is aware of how to handle that other than using git let me know and I'll gladly fix it up. I also tested without **gnome-desktop** and experienced compilation errors so I'm leaving that in for now (IRC).

Otherwise, all fixed in 23bd80cf465354b0ba859af990ea44eaf36511f2. Thanks.

Reply:
You can use its git repo but you need to use a tag then.

Reply:
The repo only has "master" and no tags or branches so I picked a commit to pin it to keep builds consistent.